class-8-js-object/bilal-yesim-js-object/script.js

```diff
@@ -0,0 +1,49 @@
+// Partners: Yeşim Nur Akar && Belal Awad
+
+// Entries should be an *array of objects*.
+const entries = [];
+
+document.querySelector(".submitButton").addEventListener('click', (e) => {
+  /*
+   * When the button is clicked, you should collect the first name,
+   * last name, and bio and store them in an object with three keys:
+   * first name, last name, and bio.
+   * 
+   * You should then add this object to the entries array.
+   */
+  
+  let obj = {
+    firstName: document.querySelector(".firstName").value,
+    lastName: document.querySelector(".lastName").value,
+    bio: document.querySelector(".bio").value,
+  };
+
+  entries.push(obj);
+  e.preventDefault();
+
+  // This function is called for you. You also have to implement it.
+  renderPeople(entries);
+});
+
+// This function should modify the element with the .output class
+// to render `people` however you want.
+//
+// THIS FUNCTION SHOULD NOT USE THE VARIABLE `entries`!
+
+function renderPeople(people) {
+  const outputDiv = document.querySelector('.output');
+  let nameSurnameP = document.createElement('p');
+  let br1 = document.createElement('br'); 
```

Comment:
why did you create 2 brs here, can't you append the one of them twice, and it would be better if you choose variable names without abbreviation

Reply:
@Ammar-64 
Actually i tried appending only one of them twice id didn't work, so i used two
The reason for two is once to seperate between the first <p>firstName lastName</p> and <p>bio</p>
and the second one is to make a break before the next two <p> tags .
i tested it with and without, it doens't break on its own, and i made them br1 and br2 while testing, didn't consider changing later sry 🤣

Reply:
yeah @awadbilal this is happening because you're reassigning the `nameSurnameP` and `bioP` variables instead of adding innerHTML or innerText to them

Reply:
@Ammar-64 shall i make new adjustments and commit them to update this file ?

Reply:
yup @awadbilal that would be better

Reply:
@Ammar-64 changes has been made, thanks for the advice ✌️